.github/workflows/release.yml

```diff
@@ -0,0 +1,107 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      publishPreRelease:
+        description: 'Publish a pre-release ?'
+        required: true
+        type: choice
+        options:
+          - 'true'
+          - 'false'
+        default: 'false'
+      publishToMarketPlace:
+        description: 'Publish to VS Code Marketplace ?'
+        required: true
+        type: choice
+        options:
+          - 'true'
+          - 'false'
+        default: 'true'
+      publishToOVSX:
+        description: 'Publish to OpenVSX Registry ?'
+        required: true
+        type: choice
+        options:
+          - 'true'
+          - 'false'
+        default: 'true'
+
+jobs:
+  packaging-job:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout vscode-yaml
+        uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          path: release
+          node-version: 20
+      - name: Install dependencies
+        run: |
+          npm install -g typescript "yarn" "@vscode/vsce" "ovsx"
+          npm install
+          echo "EXT_VERSION=$(cat package.json | jq -r .version)" >> $GITHUB_ENV
+      - name: Build vscode-yaml
+        uses: coactions/setup-xvfb@b6b4fcfb9f5a895edadc3bc76318fae0ac17c8b3 #v1.0.1
+        with:
+          run: |
+            yarn install
+            yarn run build
+            yarn run check-dependencies
+      - name: Run Unit Tests
+        uses: coactions/setup-xvfb@b6b4fcfb9f5a895edadc3bc76318fae0ac17c8b3 #v1.0.1
+        with:
+          run: yarn test --silent
+      - name: Package
+        run: |
+          node ./out/build/update-readme.js
+          vsce package -o vscode-yaml-${{ env.EXT_VERSION }}-${GITHUB_RUN_NUMBER}-${target}.vsix
```

Comment:
`${target}` isn't defined anywhere, because we package a single vsix everywhere. It's safe to remove the reference.